src/xGadgetron/cGadgetron/tests/CMakeLists.txt

```diff
@@ -67,21 +78,24 @@ target_link_libraries(MR_TESTS_CPP_AUXILIARY PUBLIC csirf cgadgetron)
 add_executable(MR_PROCESS_TESTDATA ${CMAKE_CURRENT_SOURCE_DIR}/mrtests_prep_testdata.cpp)
 add_dependencies(MR_PROCESS_TESTDATA GENERATE_SIMULATED_TESTDATA)
 target_link_libraries(MR_PROCESS_TESTDATA PUBLIC MR_TESTS_CPP_AUXILIARY csirf cgadgetron)
+set(MR_PROCESS_TESTDATA_EXE "${CMAKE_CURRENT_BINARY_DIR}/MR_PROCESS_TESTDATA")
+
 
 if(RERUN_ISMRMRD_SHEPP_LOGAN)
-        add_custom_target(PROCESS_SIMULATED_TESTDATA ALL
-                COMMAND MR_PROCESS_TESTDATA "${TESTDATA_TEMP_PREFIX}_1.h5" ${TESTDATA_OUTPUT_NAME_FULL}
-                COMMAND MR_PROCESS_TESTDATA "${TESTDATA_TEMP_PREFIX}_2.h5" ${TESTDATA_OUTPUT_NAME_GRAPPA}
-                COMMAND rm -f "${TESTDATA_TEMP_PREFIX}_1.h5"
-                COMMAND rm -f "${TESTDATA_TEMP_PREFIX}_2.h5"
-                WORKING_DIRECTORY ${TESTDATA_OUTPUT_PATH})
+  add_custom_target(PROCESS_SIMULATED_TESTDATA ALL
+          COMMAND env LD_LIBRARY_PATH=${CMAKE_INSTALL_PREFIX}/lib:$LD_LIBRARY_PATH ${MR_PROCESS_TESTDATA_EXE} "${TESTDATA_TEMP_PREFIX}_1.h5" ${TESTDATA_OUTPUT_NAME_FULL}
+          COMMAND env LD_LIBRARY_PATH=${CMAKE_INSTALL_PREFIX}/lib:$LD_LIBRARY_PATH ${MR_PROCESS_TESTDATA_EXE} "${TESTDATA_TEMP_PREFIX}_2.h5" ${TESTDATA_OUTPUT_NAME_GRAPPA}
+          COMMAND ${CMAKE_COMMAND} -E rm -f "${TESTDATA_TEMP_PREFIX}_1.h5"
+          COMMAND ${CMAKE_COMMAND} -E rm -f "${TESTDATA_TEMP_PREFIX}_2.h5"
+          WORKING_DIRECTORY ${TESTDATA_OUTPUT_PATH})
 else()
         add_custom_target(PROCESS_SIMULATED_TESTDATA ALL
         COMMAND "")
 endif()
 add_dependencies(PROCESS_SIMULATED_TESTDATA MR_PROCESS_TESTDATA)
 
 add_executable(MR_TESTS_CPLUSPLUS ${CMAKE_CURRENT_SOURCE_DIR}/mrtests.cpp)
+target_include_directories(MR_TESTS_CPLUSPLUS PUBLIC "${CMAKE_BINARY_DIR}/cmake/")
```

Comment:
I guess this is for `version.h`? But it doesn't make sense that only `MR_TESTS_CPLUSPLUS` uses it.